hack/verify-flags/verify-uniq.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
`hack/make-rules/verify.sh` isn't going to find this script because it's not in `hack/`.